index.js

```diff
@@ -14,24 +14,28 @@ const Glimmer = require('@glimmer/syntax');
 const Emblem = require('emblem').default;
 
 async function run() {
-  const NUM_STEPS = 3;
+  const NUM_STEPS = 4;
   const step = num => chalk.dim(`[${num}/${NUM_STEPS}]`);
 
   let rootDir = await pkgDir();
   let config = readConfig(rootDir);
 
   console.log(`${step(1)} 🔍  Finding JS and HBS files...`);
   let files = await findAppFiles(rootDir);
+
   console.log(`${step(2)} 🔍  Searching for translations keys in JS and HBS files...`);
   let usedTranslationKeys = await analyzeFiles(rootDir, files);
+
   console.log(`${step(3)} ⚙️   Checking for unused translations...`);
+  console.log();
+
   let translationFiles = await findTranslationFiles(rootDir);
   let existingTranslationKeys = await analyzeTranslationFiles(rootDir, translationFiles);
-
   let whitelist = config.whitelist || [];
-  let unusedTranslations = findUnusedTranslations(usedTranslationKeys, existingTranslationKeys, whitelist);
 
-  console.log();
+  let unusedTranslations = findDifferenceInTranslations(existingTranslationKeys, usedTranslationKeys, whitelist);
```

Comment:
@Turbo87 To confirm, is grouping lines 36-37 what you wanted?

After I started reordering code, I realized that you may have instead meant to ask me to show the missing translations first, then the unused translations.

Reply:
> To confirm, is grouping lines 36-37 what you wanted?

yes + the `console.log(`${step(4)} ⚙️   Checking for missing translations...`);` output too

Reply:
Cool, I see now what you wanted to see. Have updated the code.